doc/23_Installation_and_Upgrade/07_Updating_Pimcore/12_V10_to_V11.md

```diff
@@ -76,6 +76,20 @@ If you are not using Web2Print functionality, set the flag to false.
 ### WYSIWYG-Editor
 Please add all html tags with attributes you want to allow to the html_sanitizer like described [here](../../03_Documents/01_Editables/40_WYSIWYG.md)
 
+### Redis Cache Configuration
+If you are using `Redis` adapter for `Pimcore\Cache` then adapt the configuration as below:
+```yaml
+framework:
+    cache:
+        pools:
+            pimcore.cache.pool:
+                #tags: true remove it
```

Comment:
"remove it" why?

Reply:
#15625

Reply:
So it should say "remove this line when upgrading" or some such thing?

Reply:
what do you mean? it is a section in upgrade notes, basically the steps to perform before upgrading to v11.

Reply:
I don't know what I mean, that is probably because I don't really know what to do when reading these instructions. I'm basically asking for a clarification.

Reply:
I think then you got the answer in my last comment :wink:

Reply:
please follow the [discussion](https://github.com/pimcore/pimcore/issues/15605) here for the reason to remove it.

Reply:
If the discussion is key to understanding what to do here maybe it should be linked in the documentation? After all, documentation is about being explicit about what to do and not rely on subtle knowledge gleaned by combing through Github.